gallery/src/guidelines/ColorPalette/Swatcher.scss

```diff
@@ -47,3 +47,21 @@
 .gallery-swatch__hex {
   flex-basis: 7ch;
 }
+
+// Safari does not support gap in flex layouts so we need to use this query and margin for Safari.
+@media not all and (min-resolution:.001dpcm) {
+  @supports (-webkit-appearance:none) and (display:flow-root) {
+    // override a default .nx-card style
+    .nx-card__content > :first-child {
+      margin-bottom: 0;
```

Comment:
This makes me very suspicious of whether the default `.nx-card` style here is correct

Reply:
> This makes me very suspicious of whether the default `.nx-card` style here is correct

I'm overriding the Safari override styles here which is annoying. Basically in a situation where we have an `.nx-content` where the contents are stacked we want the first child to have `margin-bottom: 16px`, but not the last since that would double up the object and container margins. In situations where there are more than 2 objects stacked it's up to the dev to solve that since I was trying to avoid `.nx-card__content * { margin-bottom: $nx-spacing-md;}`.

Reply:
Here's the problem with the existing styles.  If you go into the card example on master and add a third status indicator to this card, you'll see that the margins between the indicators are different.  Since `.nx-card__content` can have `n` children, adjusting the space between only the first and second children doesn't make sense.

![image](https://user-images.githubusercontent.com/3630091/109547381-4d46c280-7a99-11eb-8601-0bd51c2261e6.png)

Reply:
> Here's the problem with the existing styles. If you go into the card example on master and add a third status indicator to this card, you'll see that the margins between the indicators are different. Since .nx-card__content can have n children, adjusting the space between only the first and second children doesn't make sense.

What I think we probably need instead is to apply a 16px bottom margin to all `.nx-card__content` children except the last one.

Reply:
> Here's the problem with the existing styles

This is now causing an issue on a downstream project:
![image](https://user-images.githubusercontent.com/3630091/109663340-fd6c0800-7b39-11eb-8f16-792acd94ab65.png)

Reply:
> > Here's the problem with the existing styles
> 
> This is now causing an issue on a downstream project:

The fix for this is here: https://github.com/sonatype/sonatype-react-shared-components/pull/324

When that is merged I'll merge Master into this branch

Reply:
> > > Here's the problem with the existing styles
> > 
> > 
> > This is now causing an issue on a downstream project:
> 
> The fix for this is here: #324
> 
> When that is merged I'll merge Master into this branch

Merged here: https://github.com/sonatype/sonatype-react-shared-components/pull/323/commits/2ab328cb98b0d5f8ac6b02a636cabb8f60b3615a